### ISSUE TITLE
Error when trying to load more with Relay v7 (Support usePagination with Relay v7)

### ISSUE THREAD

**Issue by jazblueful** [relay-tools/relay-hooks#44]
Hi Lorenzo, hope you have been doing well!

I recently upgraded from relay 6 to 7.10 and relay-hooks from 1.3.0 to 2.0.0-rc.4

Without changing any code, I now get the following error when trying to load more items using FragmentPagination:

```
Uncaught TypeError: relay_runtime_1.getFragmentOwners is not a function
    at FragmentPagination.push../node_modules/relay-hooks/lib/FragmentPagination.js.FragmentPagination._fetchPage (FragmentPagination.js:236)
    at FragmentPagination.push../node_modules/relay-hooks/lib/FragmentPagination.js.FragmentPagination.loadMore (FragmentPagination.js:224)
    at useOssFragment.js:96
...
```

I suppose getFragmentOwners function is no longer supported by relay runtime?

**Comment by morrys** [comment]
hi @jazblueful,
exactly, they are making noticeable changes within relays.
I sign this issue to look for compatibility with version 7 without losing compatibility with the 6 :)

**Comment by jazblueful** [comment]
Awesome, thank you :)

**Comment by morrys** [comment]
to check the compatibility with relay 7 I am adding the react-relay tests in the project.

  https://github.com/relay-tools/relay-hooks/commit/371ab1f532f6f1f12bd91fdcd2dd6388baf18b23

I hope to finish the new release in about ten days.

I re-open the issue to keep you updated

**Comment by morrys** [comment]
* done:
  * 155 tests of react-relay added (useMutation tests are missing)
  * verified compatibility with react-relay 7.1.0

* maybe:
  * refactor useOssFragment (too complex)

* todo:
  * test with react-relay 6.0.0

**Comment by morrys** [comment]
hello @jazblueful,
I have just released version 1.0.0-rc.6

Here you can find all the details of the changes that will be in version 1.0.0: https://github.com/relay-tools/relay-hooks/issues/45#issue-523625939

I still haven't been able to test usePagination on a real application, I wanted to ask you if you had the chance to do it :)

Thanks

**Comment by morrys** [comment]
today I have released version 2.0.0-rc.7 and also tested usePagination.

**Comment by morrys** [comment]
hi @jazblueful,
released version 2.0.0 of relay-hooks
https://github.com/relay-tools/relay-hooks/releases/tag/v2.0.0

For the moment I consider this issue closed, if you find other problems you can open it again :)

**Comment by jazblueful** [comment]
@morrys sorry for taking this long to reply, I was away.

I can confirm that it's working in a real application.

Thank you!
